realsense2_camera/launch/rs_rgbd.launch

```diff
@@ -205,12 +206,15 @@ Processing enabled by this node:
           <param name="frame_id" value="$(arg namespace)/camera_depth_optical_frame"/>
         </node>
 
+        <!-- Throttle -->
+        <node pkg="topic_tools" type="throttle" name="depth_throttler" args="messages $(arg depth)/image_rect_raw $(arg depth_throttle_rate)" />
+
```

Comment:
Should this be "image_rect_raw_throttle" instead of "image_rect_raw" ?

Reply:
no, this is correct, the first arg is the input topic